pkg/controller/storage/bucket/bucket.go

```diff
@@ -116,7 +71,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// Watch for changes to Instance
+	// Watch for changes to Bucket
```

Comment:
I've observed quite a few places in our codebase where we have questionably useful comments like this. e.g:

```go
// Do the thing to the foo
DoTheThingTo(foo)
```

Reply:
and I bet a lot of them were introduced by copying around boiler plate code :)